.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
I usually don't use .gitignore for common excludes, you can add node_modules to the global ignore file, however if you prefer I don't mind merging it anyway.